api/v1alpha1/flowcollector_types.go

```diff
@@ -224,9 +224,29 @@ type FlowCollectorConsolePlugin struct {
 	// +optional
 	Resources corev1.ResourceRequirements `json:"resources,omitempty" protobuf:"bytes,8,opt,name=resources"`
 
+	//+kubebuilder:validation:Enum=trace;debug;info;warn;error;fatal;panic
+	//+kubebuilder:default:=info
+	// LogLevel defines the log level for the console plugin
+	LogLevel string `json:"logLevel,omitempty"`
```

Comment:
Should we mention that is for console plugin `backend` side ?

Would it be also interesting to also have logs levels on frontend in the future ?

Reply:
I think we should have only one option to enable backend and frontend debug mode, but I can mention that for now this only affect backend.